gateway/modules/crud/mgo/read.go

```diff
@@ -69,8 +71,54 @@ func (m *Mongo) Read(ctx context.Context, col string, req *model.ReadRequest) (i
 			}
 		}
 
+		pipeline := make([]bson.M, 0)
+		if len(req.Aggregate) > 0 {
+			if len(req.Find) > 0 {
+				pipeline = append(pipeline, bson.M{"$match": req.Find})
+			}
+			sortFields := make([]string, 0)
+			functionsMap := make(bson.M)
```

Comment:
Does providing multiple `functions` (sum, min) in the same request work? It seems like you are overwriting the same `functionMap`?